### DISCUSSION TITLE
Permissions needed for full silent export

### DISCUSSION THREAD

**Discussion by ololax123** [Micke-K/IntuneManagement#282] [Q&A]
Hello,

I'm trying to get an full silent export working, but I seem to have problem figuring out which permissions need to be assinged to my applicaiton.

Does anyone have a list of the required permissions for read only export assignments?

**Comment by Micke-K** [comment]
Hello,

This issue list Read permissions that can be used; [Issue 263](https://github.com/Micke-K/IntuneManagement/issues/263)

Note that exporting Custom OMA-URI policies will require ReadfWrite permissions. That is a limitation in the API.

Cheers
